strings/base_types.h

```diff
@@ -85,8 +85,11 @@ WINRT_EXPORT namespace winrt
     private:
 
         template <typename TStringView>
-        static constexpr guid parse(TStringView const value)
+        static constexpr guid parse(TStringView value)
         {
+            value.remove_prefix(std::min(value.find_first_not_of(" {("), v.size()));
```

Comment:
I don't think this compiles when TStringView is std::wstring_view

Reply:
You're right. Went with the simpler option.